typing_tester.py

```diff
@@ -0,0 +1,137 @@
+import curses
+from curses import wrapper
+import time
+
+def cal_accuracy_score(given_word, user_text, no_of_backspaces):
+    """Calculates the accuracy score for typing tester"""
+
+    wrong_letter = 0
+    try:
+        wrong_letter = sum([1 for i in range(len(given_word)) if user_text[i] != given_word[i]])
+    except:
+        wrong_letter += 1
+
+    total_mistakes = wrong_letter + no_of_backspaces
+    accuracy = len(given_word) - total_mistakes
+    accuracy =  accuracy/len(given_word)*100
```

Comment:
Whats the difference between the accuracy on line 15 and 16? Always use meaningful names for variables